lib/swagger/swaggerLoader.ts

```diff
@@ -43,8 +43,8 @@ export class SwaggerLoader implements Loader<SwaggerSpec> {
     const swaggerSpec = (await (this.jsonLoader.load(specFilePath) as unknown)) as SwaggerSpec;
 
     if (this.opts.setFilePath) {
+      const pathProvider = getProviderFromSpecPath(this.fileLoader.resolvePath(specFilePath));
```

Comment:
What's the difference between this change and previous?

Reply:
It's to add support for loading swagger with relative path and uri. With previous implementation, the `specFilePath` has to be an absolute path.